openapi.yaml

```diff
@@ -25,254 +37,677 @@ security:
+    get:
+      description: "Get the ingredient with ID ingredientID"
+      tags:
+        - ingredient
+      parameters:
+        - in: path
+          name: ingredientID
+          required: true
+          type: string
+          description: ID of the ingredient to retrieve
+      responses:
+        200:
+          description: "A single ingredient"
+          schema:
+            $ref: "#/definitions/IngredientResult"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+    patch:
+      description: "Update the ingredient with ID ingredientID"
+      tags:
+        - ingredient
+      parameters:
+        - in: path
+          name: ingredientID
+          required: true
+          type: string
+          description: ID of the ingredient to retrieve
+        - in: body
+          name: attributes
+          description: attributes to be updated
+          schema:
+            $ref: "#/definitions/IngredientAttributes"
+      responses:
+        200:
+          description: "The updated ingredient"
+          schema:
+            $ref: "#/definitions/IngredientResult"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+  /ingredients/{ingredientID}/pizzas:
+    get:
+      description: "Get the pizzas made with the ingredient with ID ingredientID"
+      tags:
+        - ingredient
+        - pizza
+      parameters:
+        - in: path
+          name: ingredientID
+          required: true
+          type: string
+          description: ID of the ingredient to retrieve pizzas for
+        - in: body
+          name: filters
+          description: "Attributes to restrict pizza results to"
+          schema:
+            $ref: "#/definitions/PizzaAttributes"
+      responses:
+        200:
+          description: "A list of pizzas"
+          schema:
+            $ref: "#/definitions/PizzasResults"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
 definitions:
-  Meta:
+  PizzasResults:
     properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
-  SelfLink:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/PizzaResource"
+  PizzaResult:
     properties:
-      self:
+      data:
+        $ref: "#/definitions/PizzaResource"
+  PizzaResource:
+    properties:
+      id:
         type: string
-        format: url
-        description: Self-link of current resource
-  PaginationLinks:
+        description: a unique ID of a pizza
```

Comment:
If possible, I'd recommend giving an example for each property, it'll be super helpful for whoever use this API. Example: https://github.com/osu-mist/students-api/blob/master/openapi.yaml#L391